src/index.ts

```diff
@@ -347,6 +362,7 @@ export default function fetchHAR(har: Har, opts: FetchHAROptions = {}) {
                 } else {
                   // @ts-expect-error "Property 'from' does not exist on type 'typeof Readable'." but it does!
                   options.body = Readable.from((fileContents as File).stream());
+                  shouldSetDuplex = true;
```

Comment:
`RequestInit#duplex` should only be sent when `options.body` is a `ReadableStream`[^1], which is only these two cases where we're parsing files. The other `options.body` cases are for strings, file buffers (which aren't a stream), or the `FormData` API.

[^1]: https://fetch.spec.whatwg.org/#dom-requestinit-duplex